compose/mpp/demo/src/uikitMain/kotlin/androidx/compose/mpp/demo/main.uikit.kt

```diff
@@ -2,18 +2,37 @@
 package androidx.compose.mpp.demo
 
 import NativeModalWithNaviationExample
-import androidx.compose.runtime.remember
+import androidx.compose.runtime.*
 import androidx.compose.ui.main.defaultUIKitMain
 import androidx.compose.ui.window.ComposeUIViewController
 
 
 fun main() {
     defaultUIKitMain("ComposeDemo", ComposeUIViewController {
-        val app = remember {
-            App(extraScreens = listOf(
-                NativeModalWithNaviationExample
-            ))
-        }
-        app.Content()
+        IosDemo()
     })
 }
+
+@Composable
+fun IosDemo() {
+    var isApplicationLayout by remember { mutableStateOf(true) }
+    if (isApplicationLayout) {
+        ApplicationLayouts {
```

Comment:
Why are you moving this back to this file again? It's available from the list

Reply:
I am not found it in the list

Reply:
Sorry, my IDE didn't correctly find usages

Reply:
revert

Reply:
reverted

Reply:
Please revert this file entirely

Reply:
We still need IosDemo function, because it uses here:

https://github.com/JetBrains/compose-multiplatform-core/blob/4ddcf934d261c773a26eaad16d7267695ccba674/compose/mpp/demo-uikit/src/uikitMain/kotlin/androidx/compose/mpp/demo/getViewControllerWithCompose.kt#L24